specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2024-05-01-preview/appplatform.json

```diff
@@ -10981,6 +10981,11 @@
           "description": "Indicates whether the data plane components(log stream, app connect, remote debugging) in vnet injection instance could be accessed from internet.",
           "type": "boolean",
           "default": false
+        },
+        "enablePrivateLinks": {
```

Comment:
ARM suggests to use enum instead of bool to express a state. Like `privateLinkState: <Enabled|Disabled>`

Reply:
updated, please help review again, thx!